cmd/oomhero/main.go

```diff
@@ -17,8 +17,24 @@ var (
 
 // reads warning and critical from environment or use the default ones.
 func init() {
-	warning = envVarToUint64("WARNING", warning)
-	critical = envVarToUint64("CRITICAL", critical)
+	warningEnv := envVarToUint64("WARNING", warning)
+	criticalEnv := envVarToUint64("CRITICAL", critical)
+
+	// validate warning and critical https://github.com/ricardomaraschini/oomhero/issues/4
+	if warningEnv > 100 || criticalEnv > 100 {
+		log.Printf(
+			"warning and critical must be lower or equal to 100 .. using default values warning: %d critical: %d",
```

Comment:
As we are already logging the values being used on `main()` I think we could omit the second part of this expression. You could leave it as:

`log.Print("warning and critical must be lower or equal to 100")`

What do you think?

Reply:
makes sense .. updated